### ISSUE TITLE
Support for the stat block format often used on the sjgames forums when importing Mooks

### ISSUE THREAD

**Issue by xdy** [crnormand/gurps#106]
I *think* this is the gca statblock format.
A bunch of sample dfrpg characters were created on the sjgames forums, it would be neat if these could be imported as mooks.
https://forums.sjgames.com/showthread.php?p=2136627#post2136627

I've included a cut down sample (based on https://forums.sjgames.com/showthread.php?t=153058) below:
```
Kitharra of the Moon
Character Sheet

Sample text

More sample text.

ST 16 [60]; DX 14 [80]; IQ 10 [0]; HT 13 [30]
Damage 2d-1/3d+1; BL 51 lbs.; HP 20 [8]; Will 10 [0]; Per 13 [15]; FP 13 [0].
Basic Speed 6.25 [-10]; Basic Move 6 [0]; Move 4.
Dodge 9; Parry 12 (Spear); Block 12.

Advantages
Catfall [10]

Disadvantages and Quirks
Bloodlust (12) [-10]

Talks to "Wolf Slayer", her spear [-1]

Feature: Tail

Skills
Brawling-14 [ 1 ]
Camouflage-12* [ 1 ]

* Includes +2 for Outdoorsman

Equipment
$1,000, 77.62 lbs. (Light encumbrance)
"Wolf Slayer" (Fine Silver-coated Spear), 2d+2 imp (reach 1 or 1,2 two-handed). $200, 4 lbs.
Medium Buckler. $60, 15 lbs.
```

**Comment by crnormand** [comment]
Good idea.   I'm working on it now.   One disadvantage to that format is that they "embed" the attacks in the equipment.   But I shall try my best ;-)

**Comment by crnormand** [comment]
It is as good as it is going to get (for a while).    It has been released as part of v0.8.3
